### DISCUSSION TITLE
TreeIcon Defaults to None

### DISCUSSION THREAD

**Discussion by maheshrajrp** [eclipse-theia/theia#10985] [General] [answered]
Hi,
Earlier, TreeIcons were set to `theia-file-icons` as default. But, since it is set to `none` now, is there any way to set the default file icon to `theia-file-icons`. I tried setting via `package.json`

```
  "theia": {
    "frontend": {
      "config": {
        "applicationName": "Custom Theia",
        "preferences": {
          "workbench.iconTheme": "theia-file-icons",
          "workbench.colorTheme": "light"
        }
      }
    }
  },

```

But, this isn't being picked up by Eclipse Che. Is there anyother way to make it work or Am I missing something ?
Personally I feel like we could set `theia-file-icons` as default.

**Comment by msujew** [comment, marked as answer]
Hi @maheshrajrp,

managing the default themes (icons/colors) is not done using the preferences but purely through the config:

```json
  "theia": {
    "frontend": {
      "config": {
        "applicationName": "Custom Theia",
        "defaultIconTheme": "theia-file-icons",
        "defaultTheme": "light"
      }
    }
  },
```

> Personally I feel like we could set theia-file-icons as default.

I'm not too sure about that. With the most recent Theia update, we heavily improved the handling of contributed default icon themes. Using `none` as the default encourages downstream apps to employ their own icon themes (like the VSCode `seti` theme), instead of relying on Theia's `theia-file-icon` theme. Still, I will pick this topic up for discussion in our [next dev-call](https://github.com/eclipse-theia/theia/wiki/Dev-Meetings#agenda-2022-04-05).

**Reply by maheshrajrp** [reply]
Thanks, @msujew . Works like a charm !!
